feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/account/common/listing/delegated/DelegatedMetaAccountUpdatesListingMixin.kt

```diff
@@ -0,0 +1,26 @@
+package io.novafoundation.nova.feature_account_impl.presentation.account.common.listing.delegated
+
+import io.novafoundation.nova.feature_account_api.domain.model.LightMetaAccount
+import io.novafoundation.nova.feature_account_api.domain.model.MetaAccount
+import io.novafoundation.nova.feature_account_impl.presentation.account.common.listing.MetaAccountListingMixin
+import io.novafoundation.nova.feature_account_impl.presentation.account.common.listing.delegated.DelegatedMetaAccountUpdatesListingMixin.FilterType
+import kotlinx.coroutines.flow.Flow
+
+interface DelegatedMetaAccountUpdatesListingMixin : MetaAccountListingMixin {
+
+    sealed interface FilterType {
+        data object Proxied : FilterType
+        data object Multisig : FilterType
+        class UserIgnored(val overriddenFilter: FilterType) : FilterType
```

Comment:
IMHO it would be easier to understand if we had two flows:

1. `currentDilterType`
2. `isTypeSelectionAllowed`

As I don't see any benefits of keeping it as a special subclass case here tbh